src/renderer/lib/dolphin/useDolphinStore.ts

```diff
@@ -14,6 +14,8 @@ export const useDolphinStore = create(() => ({
   netplayOpened: false,
   playbackOpened: false,
   netplayDownloadProgress: null as { current: number; total: number } | null,
+  netplayDolphinVersion: "Not Found" as string | null,
+  playbackDolphinVersion: "Not Found" as string | null,
```

Comment:
```suggestion
  netplayDolphinVersion: null as string | null,
  playbackDolphinVersion: null as string | null,
```
These should be defaulted to `null`. You're giving more responsibility to this value than it should have. It should store the value of the version if it exists, otherwise it should be null.

What you're trying to do with distinguishing between whether Dolphin has been installed or not is already provided by the DolphinStatus. If `DolphinStatus` is UNKNOWN then that means the complete handler never completed. That would give you the "not found" status you're trying to show. Additionally the string that is shown for the version (e.g. "Not Found") should be set in the renderer component not the store.